catalyst-gateway/bin/src/settings.rs

```diff
@@ -34,14 +35,17 @@ const API_HOST_NAMES_DEFAULT: &str = "https://api.prod.projectcatalyst.io";
 /// Default `API_URL_PREFIX` used in development.
 const API_URL_PREFIX_DEFAULT: &str = "/api";
 
+/// Default docs format to be generated.
+const DEFAULT_DOCS_FORMAT: &str = "yml";
```

Comment:
No, yml is a horrible format.  Its easy to break and hard to edit.  I would rather we don't have multiple format options.  Just produce json only and make sure its correctly pretty printed and not a giant single line.

Adding features for the sake of features just multiplies the number of test cases we need for no gain.
If someone really really wants yml they can do their own conversion from json to yml.

From the ticket:
```
Even though OpenAPI can be either JSON or YAML, simply save the file as JSON. We do not need the option to save it as YAML.
```